### PR TITLE
Fix issue #166 Crash on start on device below Android Api 24.

### PR DESCRIPTION
**Pull request by TonyTangAndroid**
Fix issue https://github.com/JakeWharton/Telecine/issues/166

### CODE REVIEW

File: telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java [hunk 1]
```diff
@@ -15,7 +17,7 @@
   @Inject Analytics analytics;
 
   @Override public void onCreate() {
-    AndroidInjection.inject(this);
+    ((TelecineApplication) getApplication()).telecineComponent().api24OrGreaterServiceComponent().injector().inject(this);
```
Comment (NightlyNexus) [inline telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java]: this is unfortunate. mind if i try something to see if we can avoid the Service knowing about its injector?

Comment (NightlyNexus) [reply telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java]: opened #172 to keep the usefulness of the Android injection pieces.

Comment (TonyTangAndroid) [reply telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java]: Cool. I got you the point. And your solution is much cleaner. I have also changed the implementation in my own project. Thank you for the help.

**Comment by TonyTangAndroid** [comment]
This pull request is closed in honor of a better implementation https://github.com/JakeWharton/Telecine/pull/172.

File: telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java (hunk 1 above)
Comment (NightlyNexus) [reply telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java]: Thank you for looking into it! This was a fun problem.

**Pull request closed without merging**